WordPress/Classes/ViewRelated/Me/AccountSettingsViewController.swift

```diff
@@ -76,9 +76,17 @@ private class AccountSettingsController: SettingsController {
     func mapViewModel(_ settings: AccountSettings?, service: AccountSettingsService, presenter: ImmuTablePresenter) -> ImmuTable {
         let primarySiteName = settings.flatMap { service.primarySiteNameForSettings($0) }
 
-        let username = TextRow(
+        let username: ImmuTableRow
+        if Feature.enabled(.usernameChanging) {
+            username = EditableTextRow(
+                title: NSLocalizedString("Username", comment: "Account Settings Username label"),
+                value: settings?.username ?? "",
+                action: presenter.push(editUsername(settings)))
```

Comment:
1) The Username is not updated back on the Account Settings page.
2) On the iPad, the Username is not updated in the User Info area.

(Note, if you exit and return the Username is then updated.)